packages/expo-cli/src/commands/upload/submission-service/utils/errors.ts

```diff
@@ -0,0 +1,39 @@
+import { SubmissionError } from '../SubmissionService.types';
+import log from '../../../../log';
+
+enum SubmissionErrorCode {
+  ANDROID_UNKNOWN_ERROR = 'SUBMISSION_SERVICE_ANDROID_UNKNOWN_ERROR',
+  ANDROID_FIRST_UPLOAD_ERROR = 'SUBMISSION_SERVICE_ANDROID_FIRST_UPLOAD_ERROR',
+  ANDROID_OLD_VERSION_CODE_ERROR = 'SUBMISSION_SERVICE_ANDROID_OLD_VERSION_CODE_ERROR',
+  ANDROID_MISSING_PRIVACY_POLICY = 'SUBMISSION_SERVICE_ANDROID_MISSING_PRIVACY_POLICY',
+}
+
+const SubmissionErrorMessages: Record<SubmissionErrorCode, string> = {
```

Comment:
this is awesome, looks relatively straightforward to add to this list as we come across new cases that we want to handle